ecosystem/sep-0038.md

```diff
@@ -124,9 +124,11 @@ Name | Type | Description
 Name | Type | Description
 -----|------|------------
 `asset` | string | The [Asset Identification Format](#asset-identification-format) value.
-`delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to deliver funds to the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response.
+`sell_delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to sell/deliver funds to the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response.
+`buy_delivery_methods` | array | (optional) Only for non-Stellar assets. An array of objects describing the methods a client can use to buy/retrieve funds from the anchor. The method of delivery may affect the expiration and/or price provided in a [`POST /quote`](#post-quote) response.
+`country_codes` | array | (optional) Only for fiat assets. A list of [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) codes of the countries where the Anchor operates for fiat transactions.
 
-`delivery_methods` Object Schema
+`delivery_methods` Object Schema, used for both parameters `sell_delivery_methods` and `buy_delivery_methods`.
```

Comment:
It appears there is no `delivery_methods` anymore.
```suggestion
Object Schema for `sell_delivery_methods` and `buy_delivery_methods`.
```

Reply:
Good suggestion!